app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -58,6 +58,8 @@ def linkedin
                             reason: I18n.t('devise.linkedin.failed_to_save'))
           redirect_to after_omniauth_failure_path_for(resource_name) and return
         end
+        # Confirm user
+        @user.update_column(:confirmed_at, @user.created_at)
```

Comment:
Why are you using update_column? Just curious.

Reply:
Just to skip validations and callbacks